next/utils/xslt.ts

```diff
@@ -0,0 +1,26 @@
+// @ts-ignore
```

Comment:
so, this one isn't used and is used only for testing ? or it's something else ?

please add a comment detailing why this file exists, or remove if it's forgotten and shouldn't be here

Reply:
As could not find a declaration file for module 'saxon-js'. 
Commend added.